gtests/test_nopython_buffer.cpp

```diff
@@ -75,6 +75,103 @@ TEST(SimpleArray, abs)
     EXPECT_EQ(brr.sum(), 10.0);
 }
 
+TEST(SimpleArray, arithmetic_num)
+{
+    using namespace modmesh;
+
+#define dst_init {2, 4, 6, 8, 10, 12, 14, 16, 18, 20}
```

Comment:
It's better to follow the modmesh macro convention and name them like `MM_GTEST_???`